include/godot_cpp/variant/typed_dictionary.hpp

```diff
@@ -105,6 +120,13 @@ class TypedDictionary : public Dictionary {
 		_FORCE_INLINE_ TypedDictionary() {                                                                                 \
 			set_typed(m_variant_type, StringName(), Variant(), Variant::OBJECT, T::get_class_static(), Variant());         \
 		}                                                                                                                  \
+		_FORCE_INLINE_ TypedDictionary(std::initializer_list<KeyValue<m_type, T>> p_init) :                                \
+				Dictionary() {                                                                                             \
+			set_typed(m_variant_type, StringName(), Variant(), Variant::OBJECT, T::get_class_static(), Variant());         \
+			for (const KeyValue<m_type, T> &E : p_init) {                                                                  \
+				operator[](E.key) = E.value;                                                                               \
+			}                                                                                                              \
+		}                                                                                                                  \
```

Comment:
Strangely, `typed_dictionary.h` in Godot doesn't have this one, but I can't see any reason why. Adding this for both `TypedDictionary<T, m_type>` and `TypedDictionary<m_type, T>` seems like what we'd want to do?

@Repiteo Is missing this in Godot an oversight? Or, should this initializer list constructor really not be here?

Reply:
Raised a PR in Godot
https://github.com/godotengine/godot/pull/104664

Reply:
> Raised a PR in Godot

Thanks!